src/uint/boxed/cmp.rs

```diff
@@ -10,6 +10,52 @@ use subtle::{
     Choice, ConditionallySelectable, ConstantTimeEq, ConstantTimeGreater, ConstantTimeLess,
 };
 
+impl BoxedUint {
+    /// Return `b` if `c` is truthy, otherwise return `a`.
+    #[inline]
+    pub(crate) fn select(a: &Self, b: &Self, c: ConstChoice) -> Self {
+        debug_assert_eq!(a.limbs.len(), b.limbs.len());
+        let mut limbs = vec![Limb::ZERO; a.limbs.len()];
+
+        let mut i = 0;
+        while i < limbs.len() {
+            limbs[i] = Limb::select(a.limbs[i], b.limbs[i], c);
+            i += 1;
+        }
+
+        Self {
+            limbs: limbs.into(),
+        }
+    }
```

Comment:
This is already impl'd as [`BoxedUint::conditional_select`](https://docs.rs/crypto-bigint/0.6.0-pre.2/crypto_bigint/struct.BoxedUint.html#method.conditional_select). You can use `Into` to convert from `ConstChoice` to `Choice`.